### PR TITLE
(#997) Introduce a test to show bug

### PR DESCRIPTION
**Pull request by baudoliver7**
For #997 :

- Create `CopyBytesUntilBoundary` to extract `copy` function
- Introduce a test to show bug

### CODE REVIEW

**Review by andreoss — CHANGES_REQUESTED** [review]

File: src/test/java/org/takes/rq/multipart/CopyBytesUntilBoundaryTest.java [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+import org.junit.jupiter.api.Test;
+import org.llorllale.cactoos.matchers.Assertion;
+
+/**
+ * Test for {@link CopyBytesUntilBoundary}.
+ *
+ * @since 1.19
+ * @checkstyle MagicNumberCheck (500 lines)
+ */
+public final class CopyBytesUntilBoundaryTest {
```
[the remark this answers is not in the document]
Comment (baudoliver7) [reply src/test/java/org/takes/rq/multipart/CopyBytesUntilBoundaryTest.java]: @andreoss Done.

**Comment by baudoliver7** [comment]
@andreoss Please, see my changes.

**Review by andreoss — APPROVED** [review]

**Comment by baudoliver7** [comment]
@paulodamaso Could you check/merge ? Work has been approved by @andreoss.

**Comment by yegor256** [comment]
@rultor merge

**Comment by 0crat** [comment]
@sereshqua[/z](https://www.0crat.com/u/sereshqua) please review this job completed by @andreoss[/z](https://www.0crat.com/u/andreoss), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by sereshqua** [comment]
@andreoss please make sure you will find at least 3 issues during next CR, thanks

**Comment by sereshqua** [comment]
@0crat quality acceptable

**Comment by 0crat** [comment]
Code review was too long (137 days), architects (@paulodamaso) were penalized, see [§55](http://www.zerocracy.com/policy.html#55)

**Pull request merged**